data/src/main/java/com/teamagam/gimelgimel/data/response/entity/GeometryResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.teamagam.gimelgimel.data.response.entity;
+
+import com.teamagam.gimelgimel.data.response.entity.contents.GeoContentData;
+import com.teamagam.gimelgimel.data.response.entity.visitor.IResponseVisitor;
+
+/**
+ * A class for geo messages (using GeoContentData as its content)
+ */
+public class GeometryResponse extends GGResponse<GeoContentData> {
```

Comment:
That's not just Geometry - it's an actual message.
From the name of the class I cannot understand that this response holds the data for more than just geometry.